traits/ctraits.pyx

```diff
@@ -157,6 +157,25 @@ cdef object invalid_attribute_error():
 
     raise TypeError('Attribute name must be a string.')
 
+cdef object unknown_attribute_error(obj, name):
```

Comment:
It looks like you're adjusting the code to (roughly) follow PEP8. If that's the case, there should probably be an extra blank line above.